### PR TITLE
Feature/issue3 convert html

### PR DESCRIPTION
**Pull request by vienpn91**
Ref: #8
Sign-in flow starts from screen 1.0 to 2.2
## Todo
 - [x] Create LoginLayout & background
 - [x] Modify Ant `Header`, `Header Item` to make it match with the design
 - [x] Import and using fontawesome React
 - [x] Implement Static Login Form (3 Steps: Email --> Password --> Verification)
 - [x] Modify & override Ant `Input`, `Button`, & `Form Elements` to export our own elements for further using.
 - [ ] Modal transition - Slide from right to left when go the next step of the form
 - [x] Input placeholder transition to top when focus
 - [ ] Form error/warning displays

### CODE REVIEW

File: src/pages/login/LoginForm.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```
Comment (haunguyen90) [inline src/pages/login/LoginForm.tsx]: @vienpn91 `LoginForm.tsx` should be placed at `components` folder!!!

File: src/layouts/Header/Header.tsx [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
-        </Menu>
+        <TopMenu>
+          <MenuItem key="1">
+            <TopSearch>
+              <Icon type="search" />
+              <InputSearch placeholder="Search Here" />
+            </TopSearch>
+          </MenuItem>
+          <MenuItem key="2">
+            <FontAwesomeIcon icon={faUserCircle} size="lg" />
```
Comment (haunguyen90) [inline src/layouts/Header/Header.tsx]: @vienpn91 use https://ant.design/components/avatar/ instead

File: src/pages/login/styled.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    width: 534px;
+    border-radius: 10px;
+    .login-form-button{
+      margin-top: 10px;
+      float: right;
+      margin-bottom: 30px;
+    }
+`;
+
+export const TitleLogin = styled(Title)`
```
Comment (haunguyen90) [inline src/pages/login/styled.ts]: Does the title look the same across other forms? If yes, why don't we override the `Title` from antd instead?

File: src/pages/login/LoginForm.tsx (hunk 1 above)
Comment (vienpn91) [reply src/pages/login/LoginForm.tsx]: I fixed.

File: src/layouts/Header/Header.tsx (hunk 2 above)
Comment (vienpn91) [reply src/layouts/Header/Header.tsx]: I fixed.

File: src/pages/login/styled.ts (hunk 3 above)
Comment (vienpn91) [reply src/pages/login/styled.ts]: I fixed.

File: src/App.tsx [hunk 4]
```diff
@@ -9,23 +9,53 @@ import configureStore from './stores/configureStore';
...
+    )} />
+  );
+};
+
+const PrivateRoute: React.SFC<DefaultProps> = (props) => {
+  const { component: Component, ...rest } = props;
+
+  return (
+    <Route {...rest} render={(matchProps: RouteComponentProps) => (
+      <LoginLayout>
```
Comment (haunguyen90) [inline src/App.tsx]: @vienpn use `MainLayout` for PrivateRoute

**Pull request merged**